ansible_collections/arista/avd/plugins/action/eos_designs_documentation.py

```
@@ -4,36 +4,33 @@
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import YamlLoader, get_eos_designs_facts_path, write_file
+from ansible_collections.arista.avd.plugins.plugin_utils.utils.avd_action_plugin import AVDActionPlugin, AVDLoggingConfig
+
+if TYPE_CHECKING:
+    from pyavd._eos_designs.eos_designs_facts.schema import EosDesignsFacts
+    from pyavd._utils import get, strip_empties_from_dict
+    from pyavd.get_fabric_documentation import get_fabric_documentation
+    from pyavd.j2filters import natural_sort
 
-PLUGIN_NAME = "arista.avd.eos_designs_documentation"
 try:
     from pyavd._eos_designs.eos_designs_facts.schema import EosDesignsFacts
```

Flake8 (7.3.0)

[error] redefinition of unused 'EosDesignsFacts' from line 18

(F811)